src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+| State | $V_0$ (MeV) | $r_s$ (fm) | $a_s$ (fm) |
+|-------|-------------|------------|------------|
+| $^1S$ | -74.0       | 1.70       | 0.90       |
+| $^5S$ | -15.5       | 3.59       | 0.81       |
+| $^3P$ | -13.5       | 5.04       | 0.79       |
+| $^5D$ | -15.5       | 3.59       | 0.81       |
+
+```python
+# Parameters for nuclear potential depend on the state
+states = [
+    {"state": r"$^1S$: L=0, S=0", "L":0, "S":0, "V0": -74.0, "r_S": 1.70, "a_S": 0.90},
+    {"state": r"$^5S$: L=0, S=2", "L":0, "S":2, "V0": -15.5, "r_S": 3.59, "a_S": 0.81},
+    {"state": r"$^3P$: L=1, S=1", "L":1, "S":1, "V0": -13.5, "r_S": 5.04, "a_S": 0.79},
+    {"state": r"$^5D$: L=2, S=2", "L":2, "S":2, "V0": -15.5, "r_S": 3.59, "a_S": 0.81},
+]
+```
+
+We can then see that the attractive nuclear potentials become negligible within about 10 fm.
+
+```python
+# Generate r values (distance in fm)
+r_fm = np.linspace(0, 10, 500)
+
+# Plot the potential for each state
+plt.figure()
+for state in states:
+    V0, r_S, a_S = state["V0"], state["r_S"], state["a_S"]
+    plt.plot(r_fm, V_nuc(r_fm, V0, r_S, a_S), label=state["state"])
+
+# Customise the plot
+plt.xlabel("r (fm)", fontsize=12)
+plt.ylabel("$V_{\\rm nuc}$ (MeV)")
+plt.legend()
+plt.tight_layout()
+# plt.savefig("nuc-potential-D2.png", dpi=600)
+plt.show()
+```
+
+### Molecular potential
+
+A $\rm D_2$ molecule exists because of a balance between the electrostatic repulsion between the nuclei and an attraction  between nuclei and the electron cloud. This balance results in a 74 pm equilibrium distance between the deuterons in a gas.
+
+There are several options for representing the molecular potential. For example, the [Morse Potential](https://en.wikipedia.org/wiki/Morse_potential) is popular. For a more accurate potential, however, we will draw upon the work of [Kolos 1986](http://dx.doi.org/10.1063/1.1669836). We parameterised the numerical Kolos potential as:
+
+$$V_{mol}(r) = \frac{2}{r}(1 - b_1r - b_2r^2) e^{-\alpha r^s}$$
+
+with $r$ in units of the Bohr radius ($a_0$), $V_{mol}$ is in Rydbergs and with
+- $\alpha = 0.6255121237003474$
+- $b_1 = 1.4752572117720800$
+- $b_2 = -0.2369829512108492$
+- $s = 1.0659864120418940$
+
+We'll make a function that aligns with the nuclear potential in the sense that it takes $r$ in fm and returns the potential in MeV.
+
+```python
+alpha = 0.6255121237003474
+b1 = 1.4752572117720800
+b2 = -0.2369829512108492
+s = 1.0659864120418940
+
+# Kolos potential in MeV with r in fm
+def V_mol(r_fm):
+    r = r_fm*1e-15/a0 # fm to a0
+    return (2 / r) * (1 - b1 * r - b2 * r**2) * np.exp(-alpha * r**s) * Ry_to_eV / 1e6
+```
+
+```python
+# Range for r in pm for plotting and fm for the potential functions
+r_pm = np.linspace(25, 500, 500)
+r_fm = r_pm*1000
+
+# Calculate V_mol in MeV
+V_mol_values = V_mol(r_fm)
+
+# Plot
+plt.figure()
```

Comment:
I would also add the dashed line here at 74 pm, if for nothing else but consistency with the next plot